enterprise/views.py

```diff
@@ -1023,6 +1023,10 @@ def get(self, request):
             # Add the tpa_hint to the redirect's query parameters
             tpa_hint = enterprise_customer.identity_provider
             query_dict['tpa_hint'] = tpa_hint
+        else:
+            # Add Enterprise customer UUID and proxy_login to the redirect's query parameters
+            query_dict['enterprise_customer'] = [str(enterprise_customer.uuid)]
+            query_dict['proxy_login'] = [True]
```

Comment:
Don't we always want to add these query params?

Reply:
Yeah good catch. I assumed that for sso we'd just redirect to the tpa login screen but now that I've tested with the identity provider active I see it also goes to an edX login screen. I'll update 👍

Reply:
these are only for edX logistration to render the new sidebar, not needed for sso

Reply:
Talked with Jeff on slack and we won't need this here. Misunderstanding from a mistake in my initial proxy login PR